as-pect.config.cjs

```diff
@@ -24,5 +48,23 @@ module.exports = {
 	/**
 	 * Add your required AssemblyScript imports here.
 	 */
-	imports: {},
-}
\ No newline at end of file
+	imports(memory, createImports, instantiateSync, binary) {
+		// const imports = {env: {seed: Date.now, memory: new WebAssembly.Memory({initial: 100})}}
+		let imports = initASWebGLue({env: {seed: Date.now, memory}})
+
+		// We need to grab these methods first, otherwise as-pect's
+		// createImports function currently has an issue that will convert the
+		// functions into empty objects.
+		// https://github.com/jtenner/as-pect/issues/333
+		const {shouldLogAborts, setExports} = imports
+
+		imports = createImports(imports)
+
+		const result = instantiateSync(binary, imports)
```

Comment:
As I suggested in the pull request:

```ts
const imports = initASWebGLue({env: {seed: Date.now, memory}})
const result = instantiateSync(binary, createImports(imports))
// do whatever you expect imports to do
```

Reply:
I think your example,  

```js
const imports = initASWebGLue({env: {seed: Date.now, memory}})
const result = instantiateSync(binary, createImports(imports))

imports.setExports(result.exports)
imports.shouldLogAborts(false)
```

would work in this case, since the methods of `WebGL` of both objects refer to the same variables from a single scope created in the `initASWebGLue` call.

Also could do it very explicitly:

```js
const aspectImports = createImports({})
const webglueImports = initASWebGLue({env: {seed: Date.now, memory}})

const imports = {
  ...aspectImports,
  ...webglueImports,
  env: {
    ...aspectImports.env,
    ...webglueImports.env,
    abort() {
      webglueImports.env.abort()
      aspectImports.env.abort()
    },
  }
}

const result = instantiateSync(binary, imports)

imports.setExports(result.exports)
imports.shouldLogAborts(false)
```

Reply:
If for some reason the inner object methods rely on `this`, the two-object suggestion could break.